src/rust/madara/sources/immortalupdates/res/filters.json

```diff
@@ -0,0 +1,286 @@
+[
+	{
+		"type": "title"
+	},
+	{
+		"type": "group",
+		"name": "Status",
+		"filters": [
+			{
+				"type": "check",
+				"name": "Ongoing"
+			},
+			{
+				"type": "check",
+				"name": "On Hold"
+			},
+			{
+				"type": "check",
+				"name": "Cancelled"
+			},
+			{
+				"type": "check",
+				"name": "Completed"
+			}
+		]
+	},
+	{
+		"type": "select",
+		"name": "Genre Condition",
+		"options": [
+			"OR (having one of the selected genre)",
+			"AND (having all of the selected genres)"
+		]
+	},
+	{
+		"type": "group",
+		"name": "Genres",
+		"filters": [
+			{
+				"type": "genre",
+				"name": " Action ",
```

Comment:
why do all the genre names have spaces at the start and end? (probably from a generator script, but this should be fixed)

Reply:
fixed.